README.md

```diff
@@ -34,10 +34,9 @@ The [Developer docs](https://codaprotocol.com/docs/developers/) contain all the
 
 
 #### Quick Links:
-* [Developer readme](README-dev.md)
-* [Compiling from source and running a node](docs/demo.md)
-* [Directory structure](frontend/website/docs/developers/directory-structure.md)
-* [Lifecycle of a payment](frontend/website/docs/architecture/lifecycle-payment.md)
+* [Compiling from source and running a node](https://github.com/CodaProtocol/coda/blob/master/README-dev.md#building-coda)
```

Comment:
I think we should keep the link to `README-dev`

For the broken developers link on that page, we can substitute: https://codaprotocol.com/docs/developers

Reply:
Maybe the second link should be as before, except captioned `Running a demo node` (the developers' page doesn't really tell you how to run the compiled code)